gateway/ln-gateway/src/lib.rs

```diff
@@ -677,7 +677,7 @@ impl Gateway {
         if let GatewayState::Running {
             lnrpc,
             lightning_public_key,
-            lightning_alias: _,
```

Comment:
I was quite confused where the alias was coming from without change to the LND and CLN modules :laughing: it was there all along!

Reply:
We had it for `gateway-cli info` but never put it on the `LightningGateway` registration